test/fixtures/source_comment.rb

```diff
@@ -9,3 +9,7 @@ def bar
 # source://syntax_tree//lib/syntax_tree.rb#39
 def baz
 end
+
+# source://deleted//lib/foo.rb.rb#1
```

Comment:
Since `test/expectations/document_link/source_comment.exp.json` remains unchanged, it verifies the new behaviour is correct.